### ISSUE TITLE
[feature request] SEARCH

### ISSUE THREAD

**Issue by 0wwafa** [ip7z/7zip#61]
Please add a file search.
When opening a big archive sometimes I need just one file.
I would be  handy.

**Comment by BiatuAutMiahn** [comment]
I've been using WinRAR exclusively for it's search function. nothing else can really traverse archives for file name and/or file contents.

**Comment by callmeTayyy** [comment]
@BiatuAutMiahn I think he meant serching in the file manager. As far as I know, search is only available via command line.

**Comment by 0wwafa** [comment]
> @BiatuAutMiahn I think he meant serching in the file manager. As far as I know, search is only available via command line.

yes.

**Comment by BiatuAutMiahn** [comment]
I was talking about in the file manager too.

**Comment by callmeTayyy** [comment]
@BiatuAutMiahn not exactly, you can [list](https://7-zip.opensource.jp/chm/cmdline/commands/list.htm) the contents in the terminal and search it there:
```
7z l archive.zip
```
